jest.config.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
     '<rootDir>/node_modules/',
     'utils',
   ],
+  testTimeout: 15000,
```

Comment:
We still don't want this, right?

Reply:
yep, just a temporary solution until I can work out why they were timing out in GH Actions occasionally